test/inductor/test_templated_attention.py

```diff
@@ -169,10 +173,11 @@ def test_backwards_fails(self):
             requires_grad=True,
         )
         q, k, v = make_tensor(), make_tensor(), make_tensor()
-        out = _templated_attention(q, k, v, _identity_mod)
+        func = torch.compile(_templated_attention, backend="inductor", fullgraph=True)
         with self.assertRaisesRegex(
-            RuntimeError, "Autograd not implemented for templated_attention"
+            AssertionError, "templated_attention_backward is not an OpOverload"
```

Comment:
There is no lowering for templated_attetnion_backward yet so I suspect this is the correct failure, but I dont think this state will last for too long so hopefully this is okay for an intermediate error